usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -169,10 +168,8 @@ def get_download_ids(keyword, field, size=10000):
 
     max_iterations = 10
     total = get_total_results(keyword, '*', max_iterations)
-    if not total:
+    if total is not None:
```

Comment:
My oringinal comment might have been incorrect, we need to check if total is None for the error message